src/Uno.UI/Mixins/DependencyPropertyMixins.tt

```diff
@@ -88,7 +88,7 @@
 			.Class("Popup")
 				.Property("IsOpen", "bool", "false")
 				.Property("Child", "UIElement", "null", frameworkPropertyOption: "ValueInheritsDataContext")
-				.Property("IsLightDismissEnabled", "bool", "true")
+				.Property("IsLightDismissEnabled", "bool", "false")
```

Comment:
Why is this false and not depending on `FeatureConfiguration.EnableLightDismissByDefault`?

If a change will be made here, consider having a test for it.

Reply:
After a second thought, wouldn't this value always be ignored because of the `RegisterDefaultValueProvider` call? Or I'm misunderstanding `RegisterDefaultValueProvider` behavior?

Reply:
Yes, this value is overridden by the default value provider currently. I suppose we will be able to remove the legacy behavior altogether in the future. I used the default value provider as the `FeatureConfiguration` could be changed at any point (even runtime), so this way the default will always reflect the current setting at the point of `Popup` creation